packages/iov-bns/src/decode.spec.ts

```diff
@@ -359,5 +362,71 @@ describe("Decode", () => {
       expect(parsed.payload.chainId).toEqual("wonderland");
       expect(parsed.payload.address).toEqual("0xAABB001122DD");
     });
+
+    it("works for CreateMultisignature", () => {
+      // tslint:disable-next-line:readonly-array
+      const participants: codecImpl.multisig.IParticipant[] = [
+        {
+          signature: fromHex("1234567890"),
+          power: 4,
+        },
+        {
+          signature: fromHex("abcdef0123"),
+          power: 1,
+        },
+        {
+          signature: fromHex("9999999999"),
+          power: 1,
+        },
+      ];
+      const transactionMessage: codecImpl.app.ITx = {
+        createContractMsg: {
+          participants: participants,
+          activationThreshold: 2,
+          adminThreshold: 3,
+        },
+      };
+      const parsed = parseMsg(defaultBaseTx, transactionMessage);
+      if (!isCreateMultisignatureTx(parsed)) {
+        throw new Error("unexpected transaction kind");
+      }
+      expect(parsed.participants).toEqual(participants.map(p => p as Participant));
```

Comment:
No need to map I think: `participants as Participant[]` or `participants as readonly Participant[]`